repository/i.json

```diff
@@ -810,6 +810,17 @@
 			]
 		},
 		{
+			"name": "Ionic 2 Package",
```

Comment:
This should be changed to something like `Ionic 2 Snippets`. The word `Package` should definitely not be there.